package.json

```diff
@@ -41,16 +42,39 @@
 		"eventemitter3": "^3.1.0"
 	},
 	"devDependencies": {
-		"@types/node": "^11.13.0",
+		"@sindresorhus/tsconfig": "^0.3.0",
+		"@types/in-range": "^1.0.0",
+		"@types/node": "^11.13.2",
+		"@types/random-int": "^1.0.0",
```

Comment:
@BendingBender Can you prioritize adding type defs to `in-range` and `random-int`?

Reply:
@ltetzlaff `random-int` and `in-range` now ship with TS defs in the package, so you can drop these `@types` packages and upgrade the dependencies.

Reply:
Great 👍